data/json/items/generic.json

```
@@ -2793,6 +2793,93 @@
+        "description": "A big, body-sized pillow with a print of a blue-haired Boston-Chan, the mascot of Boston AnimeCon on both sides.  It's highly unlikely the convention would officially endorse such a lewd depiction of her.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_survivor",
+        "name": { "str": "survivor body pillow" },
+        "description": "A big, body-sized pillow with an anime-style print of Bear Grillings, Ultimate Survivor on both sides.  On one side he's holding a pneumatic air rifle, and on the other he is clad in nothing but duct tape.  The pillow is stylized to look like Kevlar.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_furry",
+        "name": { "str": "fox body pillow" },
+        "description": "A big, body-sized pillow with a print of an anthropomorphic fox on the front and a scantily-clad version on the back.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_vocaloid",
+        "name": { "str": "vocaloid body pillow" },
+        "description": "A big, body-sized pillow with a print of a popular blue-twintailed virtual musician on the front and her scantily-clad version on the back.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_goth",
+        "name": { "str": "goth body pillow" },
+        "description": "A big, body-sized pillow with a print of an anime character dressed in all black on the front and their scantily-clad version on the back.  They're surrounded by red candles and laying atop a demonic pentagram.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_yandere",
+        "name": { "str": "yandere body pillow" },
+        "description": "A big, body-sized pillow with a print of a cute maid on the front and the same maid - covered in blood and holding a knife - on the back.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_tomboy",
+        "name": { "str": "tomboy body pillow" },
+        "description": "A big, body-sized pillow with a print of a masculine woman on the front and her scantily-clad version on the back.  It's captioned with 'TOMBOY SUPREMACY' on both sides.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_femboy",
+        "name": { "str": "femboy body pillow" },
+        "description": "A big, body-sized pillow with a print of a feminine man on the front and his scantily-clad version on the back.  It's captioned with 'FEMBOY SUPREMACY' on both sides.",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_tsundere",
+        "name": { "str": "tsundere body pillow" },
+        "description": "A big, body-sized pillow with a print of a pouting anime character on the front and their scantily-clad version on the back.  They don't actually like you or anything, they swear!",
+        "weight": 1
+      },
+      {
+        "id": "bodypillow_vtuber",
+        "name": { "str": "vtuber body pillow" },
```

```suggestion
        "name": { "str": "VTuber body pillow" },
```

It's a proper noun. https://en.wikipedia.org/wiki/VTuber

(Also.... bonk 🔨)